docs/core/integration-quality-scale/rules/iqs005.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "IQS005: Support config entry unloading"
+---
+
+IQS005 is as follows:
+
+Support config entry unloading
+
+## Reasoning
+
+Sometimes a config entry ends up in a state where it is stuck.
+This can be for many reasons, but the integration may work again after it has been reloaded.
+
+In order to support this, the integration should support unloading the config entry.
+This gives users the ability to reload the integration without having to restart Home Assistant.
+Reloading an integration can be a powerful tool for users to fix issues with integrations.
+
+## Example implementation
+
+The code required to reload an integration is very little, with a lot of benefits for the user.
+This code should also be used to make sure the integration unloads correctly.
+So it should for example unsubscribe from webhooks it subscribed to or close any connections opened during the setup of the integration.
+
+The method that has to be added to `__init__.py` looks very similar to the `async_setup_entry` method.
+In this example we have a listener we want to unload, stored in the `runtime_data` of the config entry, which we want to unload to avoid memory leaks.
```

Comment:
```suggestion
In this example we have a listener, stored in the `runtime_data` of the config entry, which we want to clean up to avoid memory leaks.
```